src/entity/entity_types.cpp

```diff
@@ -24,33 +28,33 @@ Field gen_BaseEntity[] = {
     { typeid(EntityType), "type", sizeof(EntityType), (int)offsetof(BaseEntity, type) }
 };
 Field gen_Entity[] = {
+    { typeid(bool), "remove", sizeof(bool), (int)offsetof(Entity, remove) },
+    { typeid(EntityType), "type", sizeof(EntityType), (int)offsetof(Entity, type) },
     { typeid(Vec3), "position", sizeof(Vec3), (int)offsetof(Entity, position) },
     { typeid(Vec3), "scale", sizeof(Vec3), (int)offsetof(Entity, scale) },
-    { typeid(Quat), "rotation", sizeof(Quat), (int)offsetof(Entity, rotation) },
-    { typeid(bool), "remove", sizeof(bool), (int)offsetof(Entity, remove) },
-    { typeid(EntityType), "type", sizeof(EntityType), (int)offsetof(Entity, type) }
+    { typeid(Quat), "rotation", sizeof(Quat), (int)offsetof(Entity, rotation) }
```

Comment:
Maybe sort these, because there doesn't seem to be a different and I'd be best if they weren't part of the diff.